easybuild/framework/easyblock.py

```diff
@@ -1998,6 +1998,10 @@ def permissions_step(self):
             perms = stat.S_IWGRP | stat.S_IWOTH
             adjust_permissions(self.installdir, perms, add=False, recursive=True, relative=True, ignore_errors=True)
             self.log.info("Successfully removed write permissions recursively for group/other on install dir.")
+            # add read permissions for everybody on all files
```

Comment:
more like: `make sure read permissions for everyone are in place`, because usually they're already there...